docker/db-migration/migrate-develop.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
I'd propose to distinct migration for development / production envs with separate changelog roots (e.g. migrations/root.yaml and migrations/root-develop-env.yaml).

Reply:
Didn't get. How we will "move" devel migrations into prod migrations in this schema?

Reply:
There is an issue with `devel` migrations set. Some subset of queries must be executed from `root` user (to create required user and roles). If I will merge all devel migrations in one set, I had to run them from root user. I would like not to do it and keep grants for devel user and prod user the same, so we can detect lack of access on PR level and not on RC or even prod level.

Reply:
There is an issue with `devel` migrations set. Some subset of queries must be executed from `root` user (to create required user and roles). If I will merge all devel migrations in one set, I had to run them from root user. I would like not to do it and keep grants for devel user and prod user the same, so we can detect lack of access on PR level and not on RC or even prod level.